### PR TITLE
fix(core): Decrease reset password token expire time

### PR DESCRIPTION
**Pull request by tomi**
Decrease the expiration time from 1 day to 20 minutes

Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/cli/src/controllers/passwordReset.controller.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ export class PasswordResetController {
 		const resetPasswordToken = this.jwtService.signData(
 			{ sub: id },
 			{
-				expiresIn: '1d',
+				expiresIn: '20m',
```
Comment (mutdmour) [inline packages/cli/src/controllers/passwordReset.controller.ts:123]: love it.. so simple.. can you double check that the email does not say anything about a day expiry?

**Review by mutdmour — DISMISSED** [review]

File: packages/cli/src/controllers/passwordReset.controller.ts (hunk 1 above)
Comment (tomi) [reply packages/cli/src/controllers/passwordReset.controller.ts:123]: Will check 🫡

Comment (tomi) [reply packages/cli/src/controllers/passwordReset.controller.ts:123]: Nice catch! It actually said 2 hours. I amended the change to the same commit as this was so tiny change. Also rebased to master

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.15.2`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.15.2)

**Pull request merged**